### PR TITLE
Run particular playbooks for adding mon and osd

### PR DESCRIPTION
**Pull request by VasishtaShastry**
# Description

As of now we are initiating site.yml files for adding daemons
We need to use add-osd and add-mon playbooks accordingly which is being addressed by this PR  

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
NA
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
NA
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

Signed-off-by: VasishtaShastry <vipin.indiasmg@gmail.com>

### CODE REVIEW

**Comment by VasishtaShastry** [comment]
Run on 4.x cluster - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1600255937329

**Comment by rakeshgm** [comment]
@VasishtaShastry fix travis ci errors

**Comment by Ameenasuhani** [comment]
@VasishtaShastry can you create a new file  instead of making changes in tests/ceph_ansible/test_ansible_roll_over.py as this script holds good for using limit option for other daemons

**Comment by rakeshgm** [comment]
@VasishtaShastry can you rebase and make one run, This PR has new changes here https://github.com/red-hat-storage/cephci/pull/221

File: tests/ceph_ansible/test_ansible_roll_over.py [hunk 1]
```diff
@@ -129,10 +129,27 @@ def run(ceph_cluster, **kw):
 
     log.info("Ceph versions " + ceph_installer.get_installed_ceph_versions())
 
+    # Run site-x.yml by default
+    if ceph_cluster.ansible_config.get('containerized_deployment'):
```
Comment (sunilkumarn417) [inline tests/ceph_ansible/test_ansible_roll_over.py:133]: Is site-container.yaml will be considered here?

**Review by sunilkumarn417 — CHANGES_REQUESTED** [review]

File: tests/ceph_ansible/test_ansible_roll_over.py [hunk 2]
```diff
@@ -129,10 +129,27 @@ def run(ceph_cluster, **kw):
...
+        yaml_file = "site-docker.yml"
+    else:
+        yaml_file = "site.yml"
+
+    # Run add-mon.yml if mon is to be added and build is 4.x
```
Comment (sunilkumarn417) [inline tests/ceph_ansible/test_ansible_roll_over.py:138]: does this also requires inventory file update?

**Review by sunilkumarn417 — CHANGES_REQUESTED** [review]

File: tests/ceph_ansible/test_ansible_roll_over.py (hunk 1 above)
Comment (VasishtaShastry) [reply tests/ceph_ansible/test_ansible_roll_over.py:133]: Thanks for the heads-up
Changed accordingly

**Comment by rakeshgm** [comment]
are anymore updates needed here ?

**Comment by Ameenasuhani** [comment]
fixed in https://github.com/red-hat-storage/cephci/pull/269

**Pull request closed without merging**